core/controllers/base.py

```diff
@@ -70,6 +70,27 @@ def get(self):
         auth_services.establish_auth_session(self.request, self.response)
 
 
+class SessionEndHandler(webapp2.RequestHandler):
+    """Class which handles the creation of a new authentication session."""
+
+    def get(self):
+        """Establishes a new auth session."""
+        auth_services.destroy_auth_session(self.response)
+
+
+class CreateInitialSuperAdmin(webapp2.RequestHandler):
```

Comment:
Why does this need to be in base.py?

Reply:
See above -- it is temporary in nature and will be removed, and doesn't correspond neatly to any other category.

(Though maybe @brianrodri it's worth adding a TODO and issue at least.)

Reply:
Done